docs/docs/reference/cli/init.mdx

```diff
@@ -3,16 +3,16 @@ title: 'init'
 sidebar_position: 1
 ---
 
-Initialize a local Nhost project in the current working directory.
+Initialize a local Nhost project.
 
 ```
 nhost init
 ```
 
 If you already have a Nhost project in Nhost Cloud you can use that project as a starting point by appending `--remote` to the command.
 
-This will pull the database migrations and Hasura metadata from the Nhost Cloud project locally for you to use as a starting point.
-
 ```
 nhost init --remote
 ```
+
+This will reset and pull the database migrations and Hasura metadata from the Nhost Cloud project locally for you to use as a starting point. While doing `nhost init --remote` it's also recommended to setup [Git](/platform/git) to activate automated deplouments on `git push` for your Nhost project.
```

Comment:
```suggestion
This will reset the local database migrations and Hasura metadata, and pull the database migrations and Hasura metadata from the Nhost Cloud project locally for you to use as a starting point. While doing `nhost init --remote` it's also recommended to setup [Git](/platform/git) to activate automated deplouments on `git push` for your Nhost project.
```